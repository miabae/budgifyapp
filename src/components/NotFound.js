import React from 'react';
import { Link } from 'react-router-dom';

const NotFound = () => (
    <div>
        404 Error - Go <Link to="/">home</Link>.
    </div>
);

export default NotFound;