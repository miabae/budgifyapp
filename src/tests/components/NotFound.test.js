import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../components/NotFound';

test('should render Not Found page correctly', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
});