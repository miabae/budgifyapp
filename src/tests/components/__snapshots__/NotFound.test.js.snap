// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Not Found page correctly 1`] = `
<div>
  404 Error - Go 
  <Link
    replace={false}
    to="/"
  >
    home
  </Link>
  .
</div>
`;
